include/mbedtls/platform_util.h

```diff
@@ -60,6 +60,24 @@ MBEDTLS_DEPRECATED typedef int mbedtls_deprecated_numeric_constant_t;
 #endif /* MBEDTLS_DEPRECATED_WARNING */
 #endif /* MBEDTLS_DEPRECATED_REMOVED */
 
+/** \def MBEDTLS_CHECK_RETURN
+ *
+ * This macro appearing at the beginning of the declaration of a function
+ * indicates that its return value should be checked.
+ *
+ * This should appear before most functions returning an error code
+ * (as \c int in the \c mbedtls_xxx API or
+ * as ::psa_status_t in the \c psa_xxx API).
+ */
+#if defined(__GNUC__)
```

Comment:
I think as Gilles said, this should be guarded by `#if !defined(MBEDTLS_CHECK_RETURN)`, in order to allow users to provide another definition in the user config file.

Reply:
force pushed with fixed ifdef